vendor/assets/javascripts/data-confirm-modal.js

```diff
@@ -137,6 +137,16 @@
       '</div>'
     );
 
+    // Make sure it's always the top zindex
+    var highest = 1050;
```

Comment:
1050 is what bootstrap modals z-index is set at by default, so if you want the modal from the confirm to show up above any other manipulated modals, it has to have the highest z-index.